packages/mask/src/extension/popups/components/LoadingPlaceholder/index.tsx

```diff
@@ -15,15 +15,21 @@ const useStyles = makeStyles()((theme) => ({
     },
 }))
 
-export const LoadingPlaceholder = memo(() => {
+export interface LoadingPlaceholderProps {
+    title?: string
+    titleColor?: string
+    iconColor?: string
+}
+
+export const LoadingPlaceholder = memo((props: LoadingPlaceholderProps) => {
     const { t } = useI18N()
     const { classes } = useStyles()
 
     return (
         <main className={classes.container}>
-            <LoadingIcon style={{ color: '#1C68F3' }} />
-            <Typography variant="caption" color="#A6A9B6">
-                {t('loading')}
+            <LoadingIcon style={{ color: props.iconColor ?? '#1C68F3' }} />
```

Comment:
```
theme.palette.public.primary // '#1C68F3'
```

Reply:
@nuanyang233 Can I use it?

Reply:
We don't override the theme on the popup right now, so I suggest using the dead code color first.